### ISSUE TITLE
windows速度

### ISSUE THREAD

**Issue by cacard** [andimarafioti/faster-qwen3-tts#97]
直接使用 .venv 环境， cuda12.8， win11 + 5090laptop 24gvarm

benchmark result:
=== Faster Qwen3-TTS Benchmark ===
GPU: NVIDIA GeForce RTX 5090 Laptop GPU
PyTorch: 2.11.0+cu128
CUDA: 12.8

--- Benchmarking 0.6B ---
Loading model...

`torch_dtype` is deprecated! Use `dtype` instead!
Fetching 4 files: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<?, ?it/s]
Could not infer sample rate from base model; defaulting to 24000 Hz.

Warmup run (includes CUDA graph capture)...
Warming up predictor (3 runs)...
Capturing CUDA graph for predictor...
CUDA graph captured!
Warming up talker graph (3 runs)...
Capturing CUDA graph for talker decode...
Talker CUDA graph captured!
Warmup: 6.52s

Measuring streaming TTFA (5 runs per chunk size)...
  chunk_size= 4: TTFA=167ms ± 6ms
  Run 1: 244.2ms
  Run 2: 235.0ms
  Run 3: 235.1ms
  Run 4: 235.3ms
  Run 5: 237.9ms
  chunk_size= 8: TTFA=237ms ± 4ms <<
  chunk_size=12: TTFA=314ms ± 5ms

Streaming baseline vs fast path (chunk_size=8)...
  baseline: TTFA=1217ms ± 50ms, RTF=0.568 ± 0.003
  fast:     TTFA=236ms ± 2ms, RTF=3.606 ± 0.055

Saved sample audio to D:\_AI\tts_qwen3_faster\faster-qwen3-tts\sample_0.6B.wav
Results saved to bench_results_NVIDIA_GeForce_RTX_5090_Laptop_GPU.json

--- Benchmarking 1.7B ---
Loading model...

`torch_dtype` is deprecated! Use `dtype` instead!
Fetching 4 files: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4/4 [00:00<?, ?it/s]
Could not infer sample rate from base model; defaulting to 24000 Hz.

Warmup run (includes CUDA graph capture)...
Warming up predictor (3 runs)...
Capturing CUDA graph for predictor...
CUDA graph captured!
Warming up talker graph (3 runs)...
Capturing CUDA graph for talker decode...
Talker CUDA graph captured!
Warmup: 6.60s

Measuring streaming TTFA (5 runs per chunk size)...
  chunk_size= 4: TTFA=178ms ± 4ms
  Run 1: 266.3ms
  Run 2: 259.4ms
  Run 3: 259.2ms
  Run 4: 262.2ms
  Run 5: 258.7ms
  chunk_size= 8: TTFA=261ms ± 3ms <<
  chunk_size=12: TTFA=341ms ± 4ms

Streaming baseline vs fast path (chunk_size=8)...
  baseline: TTFA=1237ms ± 51ms, RTF=0.543 ± 0.005
  fast:     TTFA=257ms ± 1ms, RTF=3.180 ± 0.021

Saved sample audio to D:\_AI\tts_qwen3_faster\faster-qwen3-tts\sample_1.7B.wav
Results saved to bench_results_NVIDIA_GeForce_RTX_5090_Laptop_GPU.json

Done. Results saved as bench_results_*.json

**Comment by cacard** [comment]
我用单独代码推理，0.6b模型生成6秒音频需要7.5秒？这就不快了啊

**Comment by andimarafioti** [comment]
如果 benchmark 已经能跑到这个速度，那说明你这台机器肯定是可以达到的。你在单独代码里如果只跑到这么低的速度，那基本可以确定是你的调用方式或者代码哪里写错了。

请把你实际推理用的代码片段贴出来，不然我这边没法进一步帮你定位问题。

**Comment by cacard** [comment]
`

model = FasterQwen3TTS.from_pretrained("Qwen/Qwen3-TTS-12Hz-0.6B-Base") # 1.7B 0.6B < ————————————————————————————————————————————————————
ref_audio = "ref_audio.wav"
ref_audio = 'l.wav'
ref_text = (
    "I'm confused why some people have super short timelines, yet at the same time are bullish on scaling up "
    "reinforcement learning atop LLMs. If we're actually close to a human-like learner, then this whole approach "
    "of training on verifiable outcomes is doomed."
)
ref_text = ("大家好，我是这是一些例子。我是这是一些例子。我是这是一些例子。")

start = time.perf_counter()
audio_list, sr = model.generate_voice_clone(
    text="把 numpy 音频数据 → 转成 WAV 格式 → 再编码成 Base64 字符串", language="Chinese", # English
    ref_audio=ref_audio, ref_text=ref_text,
)
cost = time.perf_counter() - start
print(f'Cost: {cost:.2f}s')
audio = _concat_audio(audio_list)
_to_wav_and_save_file(audio, sr, "output.wav")


`

**Comment by andimarafioti** [comment]
You're mixing two different things. Generating the voice clone itself takes some time on top of generating the audio. But you can do it once and save it. The readme should be clear about how to achieve that.
